apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1764,6 +1764,14 @@ public static Bundle createBundle(
        * TODO: Above bug should be fixed as of 1/1/20; re-investigate this
        */
       int endIndex = Math.min(paging.getStartIndex() + paging.getPageSize(), resources.size());
+      // Throw a 400 if startIndex >= results, since we cant sublist with these values
+      if (paging.getStartIndex() >= resources.size()) {
+        throw new InvalidRequestException(
+            String.format(
+                "Value for startIndex (%d) must be less than than result size (%d)",
+                paging.getStartIndex(), resources.size()));
+      }
```

Comment:
This is the crux of the fix that affects all endpoints' bundling; everything calls this method to do bundling (except PAC, which does it in the abstract resource provider directly)

Reply:
Can a client of the API know before making a request what the `resources.size()` will be?  If not is this really a client error?  Or just an edge case where we should return an empty response?

Reply:
I considered this a bit when designing the fix, whether to just do some default behavior like returning no results.

1. They wont necessarily know ahead of time if they are making their first call, but an empty response isnt as clear that there were results and with a fixed input they would return, and I'd rather indicate there was a potential issue with the nonsensical input.
2. Generally startIndex is used with a query where the result amount _should_ be known by having made a call first, otherwise why are they indexing into the results?